pcsx2/GameList.cpp

```diff
@@ -675,13 +678,24 @@ bool GameList::ScanFile(
 		return true;
 	}
 
-	auto iter = played_time_map.find(entry.serial);
+	const auto iter = played_time_map.find(entry.serial);
 	if (iter != played_time_map.end())
 	{
 		entry.last_played_time = iter->second.last_played_time;
 		entry.total_played_time = iter->second.total_played_time;
 	}
 
+	auto custom_title = custom_attributes_ini.GetOptionalStringValue(entry.path.c_str(), "Title");
+	if (custom_title)
+	{
+		entry.title = std::move(*custom_title);
```

Comment:
Nitpick, but I tended to use `optional.value()`, rather than `*optional`, so best to be consistent.